src/test/java/wbooks/userdata/userdata.feature

```diff
@@ -0,0 +1,42 @@
+Feature: get data from specific user
+
+  Background:
+    * url 'https://wbooks-api-stage.herokuapp.com/api/v1'
+
+    * def login = callonce read('classpath:wbooks/login/login.feature@login')
+    * def authToken = login.access_token
+
+    * def userDataSchema =
+    """
+    {
+      id: '#number',
+      email: '#string',
+      last_name: '#string',
+      first_name: '#string',
+      rents_counter: '#number',
+      comments_counter: '#number',
+      image_url: '##string'
+    }
+    """
+
+  Scenario: get user data
+    * def userId = 2
+    Given path 'users/' + userId
```

Comment:
use this way to define the path: `Given path 'users' userId`

Reply:
Moved to background.
I don't like it much because now the background has even more variables that are scenario specific, but that might be more of a problem of how the feature is set (expecting all the different /users/:id status codes)